lib/bozo/compilers/msbuild.rb

```diff
@@ -184,10 +188,36 @@ def framework_version
       framework_version
     end
 
+    def windows_path
+      to_unix_path(ENV['WINDIR'])
+    end
+
+    def default_msbuild_path(config)
+      File.join(windows_path, 'Microsoft.NET', config[:framework], config[:version], 'msbuild.exe')
+    end
+
+    def program_files
+      globalArch = ENV['PROCESSOR_ARCHITECTURE']
```

Comment:
You're right.  But not because of this line I think.  We need to take another look.